myFirstWebsite/css/styles.css

```diff
@@ -158,33 +318,484 @@ body {
 .benefits__item {
     max-width: 300px;
     margin-bottom: 119px;
+    display: flex;
+    flex-direction: column;
 }
 
 .benefits__item-title {
     font-weight: 500;
     font-size: 18px;
     line-height: 24px;
-    color: #000000;
+    color: var(--text-color-main);
     margin: 37px 0 26px 0;
 }
 
 .benefits__item-description{
     font-weight: normal;
     font-size: 16px;
     line-height: 24px;
-    color: #000000;
+    color: var(--text-color-main);
     padding-bottom: 39px;
 }
 
 .benefits__item-more-link {
+    margin-top: auto;
+    padding-right: 46px;
+    max-width: 150px;
+}
+
+/* Benefit ^^/
+
+/* Locations */
+
+.wrapper__main {
+    max-width: 1262px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+
+.locations__wrapper {
+    max-width: 1262px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+
+.locations__title-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 36px;
+}
+
+.locations__items {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-gap: 10px;
+    margin-bottom: 131px;
+}
+
+.locations__item {
+    position: relative;
+}
+
+.locations__item-title {
+    font-size: 50px;
+    line-height: 60px;
+    color: var(--common-title-color);
+    position: absolute;
+    bottom: 27px;
+    left: 29px;
+}
+
+.locations__item-big {
+    grid-column: span 3;
+}
+
+.locations__item-small {
+    grid-column: span 2;
+}
+
+.locations__item-image {
+    width: 100%;
+    object-fit: cover;
+}
+
+.locations__item-big > img {
```

Comment:
Get rid of nesting selectors. Look in BEM how to do it

Reply:
+